packages/web/src/pages/sign-up-page/pages/SelectArtistsPage.tsx

```diff
@@ -166,6 +175,10 @@ export const SelectArtistsPage = () => {
                   css={{ minHeight: 500 }}
                   direction='column'
                 >
+                  <HiddenLegend>
```

Comment:
What is this for?

Reply:
its for a11y, where the checklist fieldset has a label, which comes from a <legend> vs aria-label